jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/EntityManager.java

```diff
@@ -262,7 +264,32 @@ private void prepareDocs(Map<String, Map<String, String>> currentInfo, List<Map<
             } else if (entityInfo.containsKey("projectId")) {
                 entityInfo.put("accountid", entityInfo.get("projectId"));
             }
+            //For GCP CQ Collector accountName will be fetched from RDS using accountId
```

Comment:
@AnandPaladin we would know that data source at the beginning of the job. Let's cache the DB account name request once at the beginning of the job.

Reply:
@plyubich I have used HashMap which act as **InMemorycache** to check whether it already contains {projectId,project Name} .Only if it does not contain entry then only we are making RDS call. Please clarify this is not serving the purpose as that of cache ?

CC:@mkinchin-paladin